Uplift/Views/Cells/FacilitiesDropdownCollectionViewCell.swift

```diff
@@ -0,0 +1,168 @@
+//
+//  FacilitiesDropdownCollectionViewCell.swift
+//  
+//
+//  Created by Cameron Hamidi on 11/6/19.
+//
+
+import UIKit
+
+class FacilitiesDropdownCollectionViewCell: UICollectionViewCell {
+    var collectionView: UICollectionView!
+    var collectionViewHeight: CGFloat!
+    var dropdownView: DropdownView!
+    var facility: Facility!
+    var headerImageView: UIImageView!
+    var headerNameLabel: UILabel!
+    var headerOpenLabel: UILabel!
+    var headerView: UIView!
+    var headerViewHeight: CGFloat = 52
+    var height: CGFloat!
```

Comment:
can we make these `private let` and provide default values where possible instead of using `!`? i.e. the UILabels, UIViews, CGFloats